packages/wonder-blocks-modal/src/components/__tests__/modal-launcher.test.js

```diff
@@ -284,50 +294,140 @@ describe("ModalLauncher", () => {
         await wait();
 
         // Assert
+        // eslint-disable-next-line testing-library/no-node-access
         expect(document.activeElement).not.toBe(lastButton);
     });
 
     test("if modal is closed, return focus to the last element focused outside the modal", async () => {
```

Comment:
@jandrade This is the test I updated for the coverage test to stop failing!

Reply:
Thanks for fixing this test! the solution is great!